app/scripts/components/common/card/horizontal-info-card.tsx

```diff
@@ -70,36 +69,27 @@ interface Props {
 }
 
 export default function HorizontalInfoCard(props: Props) {
-    const {
-    title,
-    description,
-    imgSrc,
-    imgAlt,
-    tagLabels,
-  } = props;
+  const { title, description, imgSrc, imgAlt, tagLabels } = props;
 
   return (
     <HorizontalCard>
       <CardImage>
-        <img src={imgSrc} alt={imgAlt} loading='lazy' />
+        {imgSrc && <img src={imgSrc} alt={imgAlt} loading='lazy' />}
```

Comment:
I am wondering if we should wrap the whole `<CardImage />` component in the conditional? Seems unnecessary to just keep an empty component, when there is no `imgSrc`

Reply:
Good point @AliceR. I initially wrapped the entire `<CardImage />` component but I want to avoid collapsing the layout entirely when there’s no `imgSrc`. So I kept the styled component to display a fallback blue-ish square.